src/routes/safe/components/Apps/components/AppsList.tsx

```diff
@@ -110,40 +120,50 @@ const AppsList = (): React.ReactElement => {
         </Col>
       </Menu>
       <ContentWrapper>
-        <CardsWrapper>
-          <AppCard iconUrl={AddAppIcon} onClick={openAddAppModal} buttonText="Add custom app" iconSize="lg" />
-
-          {appList
-            .filter((a) => a.fetchStatus !== SAFE_APP_FETCH_STATUS.ERROR)
-            .map((a) => (
-              <AppContainer key={a.url}>
-                <StyledLink key={a.url} to={`${appsPath}?appUrl=${encodeURI(a.url)}`}>
-                  <AppCard isLoading={isAppLoading(a)} iconUrl={a.iconUrl} name={a.name} description={a.description} />
-                </StyledLink>
-                {isCustomApp(a.url, appList) && (
-                  <IconBtn
-                    title="Remove"
-                    onClick={(e) => {
-                      e.stopPropagation()
-
-                      setAppToRemove(a)
-                    }}
-                  >
-                    <Icon size="sm" type="delete" color="error" />
-                  </IconBtn>
-                )}
-              </AppContainer>
-            ))}
-        </CardsWrapper>
-
-        <IconText
-          color="secondary"
-          iconSize="sm"
-          iconType="info"
-          text="These are third-party apps, which means they are not owned, controlled, maintained or audited by Gnosis. Interacting with the apps is at your own risk. Any communication within the Apps is for informational purposes only and must not be construed as investment advice to engage in any transaction."
-          textSize="sm"
-        />
+        <SearchCard value={appSearch} onValueChange={(value) => setAppSearch(value.replace(/\s{2,}/g, ' '))} />
```

Comment:
Should this be done inside the SearchCard component itself maybe? `value.trim()` would probably not hurt either.

I would also call the component `SearchInput`.

Reply:
> value.trim() would probably not hurt either.

Been there, if the value is trimmed user will be forced to use only one-word searches. Trimming before passing it to the fuse.js may work. I'll check if it makes sense and affects the search results

Tests are on the way; it wasn't ready for another review round 😛

Reply:
I tested, and the space at the end doesn't affect the search results, so I'll resolve this comment

I will rename the component